cli/src/swam/cli/Main.scala

```diff
@@ -77,6 +59,9 @@ object Main extends CommandIOApp(name = "swam-cli", header = "Swam from the comm
   val trace =
     Opts.flag("trace", "Trace WASM execution channels (default false)", short = "t").orFalse
 
+  val cov =
+    Opts.flag("instcov", "Run the WebAssembly module and gets coverage.", short = "v").orFalse
```

Comment:
@tareq97 I think this is redundant since you are calling coverage from cli.run

Reply:
@Jacarte Actually I added this because I was not sure if coverage command will be used separately or run command. If it is not used with the run command we can remove this option. But If coverage need to be added to run command then we can use this option to generate the coverage or not. This point needs to be discussed.